flume/neore/scripts/package.sh

```diff
@@ -98,14 +99,7 @@ function copy_cygnus_conf() {
 	_logStage "######## Copying cygnus template config files to destination config directory... ########"
 	rm -rf {RPM_SOURCE_DIR}/config 
 	mkdir -p ${RPM_SOURCE_DIR}/config
-	for file in $(ls ${BASE_DIR}/conf)
-	do
-		# file=$(basename ${file})
-		# renamed_file=${file%.template}
-		# cp ${BASE_DIR}/conf/${file} ${RPM_SOURCE_DIR}/config/${renamed_file}
-
-		cp ${BASE_DIR}/conf/${file} ${RPM_SOURCE_DIR}/config/
-	done
+        cp ${BASE_DIR}/conf/* ${RPM_SOURCE_DIR}/config/
```

Comment:
In addition, github diff shows one whitespace offset in the indentation also here...